source/blog/2016-01-25-ADB-1.6-announcement.html.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Atomic Developer Bundle v1.6 Released
+author: bexelbie
+date: 2016-01-24 11:50:00 UTC
+tags: vagrant, Docker, Developer tools, openshift, kubernetes, centos
+published: false
+comments: true
+---
+Project Atomic is pleased to announce the release of the Atomic Developer Bundle (ADB) 1.6, a Vagrant box that provides a ready-to-use development environment for container applications.  With the ADB, developers can dive right into producing complex, multi-container applications.  The ADB is useful on Microsoft Windows, Apple's OS X and GNU/Linux distributions.
+
+The 1.6 release adds several new features, including:
+
+  * [Mesos Marathon](https://mesosphere.github.io/marathon/) as a supported container orchestrator
+  * A new delivery location in HashiCorp's Atlas box catalog, [projectatomic/adb](https://atlas.hashicorp.com/projectatomic/boxes/adb)
+  * Updated Documentation, a new Maintainer and [more](https://github.com/projectatomic/adb-atomic-developer-bundle/releases/tag/v1.6.0)
+
+The ADB provides a container development ecosystem that ensures that developers are able to work with Linux containers even if their base operating system does not support it.
+
+READMORE
+
+The ADB provides a quick and easy way to setup multi-container applications that take advantage of management and/or orchestration solutions, such as OpenShift or [Kubernetes](http://kubernetes.io/). The ADB also supports [Nulecule](http://www.projectatomic.io/docs/nulecule/) and [Atomic App](http://www.projectatomic.io/docs/atomicapp/). These projects provide a structure for packaging, distributing, and deploying multi-container applications. 
```

Comment:
a hyperlink to OpenShift should also help, as we are hyperlinking Kubernetes or Nulecule, etc.

[OpenShift] (https://www.openshift.com/)